docs/resource-sharing/os-backfill-containers.md

```diff
@@ -94,6 +95,15 @@ but it has other advantages such as supporting automounting of repositories.
 
 ### Adding CVMFS using cvmfsexec
 
+!!! important "EL7 hosts require an updated kernel"
+    EL7 hosts must have kernel version >= 3.10.0-1127 (run `uname -vr` to check) with user namespaces enabled and
+    network namespaces disabled.
+    See this section on using
+    [unprivileged Singularity](https://opensciencegrid.org/docs/worker-node/install-singularity/#enabling-unprivileged-singularity)
+    for details.
+
+    See the [cvmfsexec README](https://github.com/cvmfs/cvmfsexec#readme) details.
```

Comment:
This is good enough but I would say "for additional details" instead of "details".

Reply:
Done!

Reply:
Actually, you changed the previous line's details, not this one.

Reply:
I ended up tossing the referenced sentence altogether. Especially since pointed out that the unpriv user namespaces apply to both bind-mounted CVMFS + cvmfsexec cases. We also already reference it in the first mention of `cvmfsexec` in the cvmfsexec section.